src/Components/LabelContainer.jsx

```diff
@@ -1,45 +1,122 @@
 import React from "react";
-import { Container, Table} from 'react-bootstrap'
+import { Container, Table, Dropdown } from 'react-bootstrap'
 import Label from './Label'
+import './label.css'
 
 
 import testData from './testData.json'
 
+let tempVar = 0
 
 
 class LabelContainer extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      
+      completeUnits: true
     };
+
+    this.renderCompleteUnits = this.renderCompleteUnits.bind(this);
+    this.renderTableSpoonUnits = this.renderTableSpoonUnits.bind(this);
+    this.completeUnitsButton = this.completeUnitsButton.bind(this);
+    this.tableSpoonUnitsButton = this.tableSpoonUnitsButton.bind(this);
+  }
+
+  completeUnitsButton(){
+    this.setState({ completeUnits: true})
+  }
+
+  tableSpoonUnitsButton(){
+    this.setState({ completeUnits: false})
+  }
+
+
+  renderCompleteUnits() {
+    if (this.state.completeUnits===true){
+      return (
+        <div> <p className="text-center">Nutrients for 100 {testData[0].servingSizeUnit} </p>
+        <Table responsive>
+        <thead>
+            <tr>
+            <th>Item</th>
+            <th><i>Quantity</i></th>
+            <th><i>Units</i></th>
+            </tr>
+        </thead>
+        
+        {/*Double map: first loop maps the testData json file and second loop goes through the 'foodNutrients' array.*/}
+        
+        {testData.map((value, index) => {
+               return (
+                value.foodNutrients.map((value, index) => (
+                  //testing logs
+                  //console.log(varName), //console.log(value.nutrient.name), //console.log(testData[0]), //console.log(value.nutrient.name), //console.log(value.amount),
+                  console.log(testData[0]),
+                    <Label names={value.nutrient.name} values={Math.round(value.amount).toFixed(1)} units={value.nutrient.unitName}/>
+                ))
+               )
+        })}
+       </Table>
+       </div>
+      )
+    }
+  }
+
+  renderTableSpoonUnits() {
+    if (this.state.completeUnits===false){
+      return (
+        <div> <p className="text-center">Nutrients for {testData[0].householdServingFullText} ( {testData[0].servingSize} {testData[0].servingSizeUnit} ) </p> 
+        <Table responsive>
+        <thead>
+            <tr>
+            <th>Item</th>
+            <th><i>Quantity</i></th>
+            <th><i>Units</i></th>
+            </tr>
+        </thead>
+        
+        {/*Double map: first loop maps the testData json file and second loop goes through the 'foodNutrients' array.*/}
+        
+        {testData.map((value, index) => {
+               return (
+                value.foodNutrients.map((value, index) => (
```

Comment:
This function is wrapping using parenthesis instead of curly brackets. Please replace this on Lines 82 and 88